pages/common/aws-cur.md

```diff
@@ -0,0 +1,16 @@
+# aws cur
+
+> Create, query, and delete AWS usage report definitions.
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/cur/index.html>.
+
+- Create an AWS cost and usage report definition from a JSON file:
+
+`aws cur put-report-definition --report-definition file://{{path/to/report-definition.json}}`
```

Comment:
I think we can use the samba URI too `smb://path/to...` so:
```suggestion
`aws cur put-report-definition --report-definition {{file://path/to/report-definition.json}}`
```

Reply:
Tried using smb on a machine with samba installed and it didn't work. Have you tried using that notation with this command?  Can you share the details so I can replicate it?

Reply:
@navarroaxel I'm not so sure about your suggestion. If you have to use `file://` every time, it's not really a variable. It's the same with quotes, where we also use `"{{text}}"`.

Reply:
@bl-ue  and I haven't been able to get  `smb://path/to...` to work with aws. At this point I don't think it is compatible.

Reply:
`smb:` has a different syntax anyway, `smb://<host>/<path>` (I believe)

Reply:
Perhaps someone with a numeric name should make a page for smbd :stuck_out_tongue_closed_eyes:

Reply:
Ok, if `smb://` doesn't work this is ok then.

Reply:
I'll spend some more time experimenting with smb and see if I can find a usage that will work.

Reply:
If it becomes too much work, forget it @285204, because it's likely that only a very few persons will even read this, let alone try it out for real.

Reply:
I did a little bit more and convinced myself it doesn't work.

Reply:
It's a no-go then.  Let's not document it.